### PR TITLE
Update and rename Ahnlab.com.xml to AhnLab.com.xml

### PR DESCRIPTION
**Pull request by pipboy96**
No evidence of mixed content issues.

### CODE REVIEW

File: src/chrome/content/rules/AhnLab.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+	Connection refused:
+		- ^
```
Comment (Bisaloo) [inline src/chrome/content/rules/AhnLab.com.xml]: download

Comment (pipboy96) [reply src/chrome/content/rules/AhnLab.com.xml]: @Bisaloo Done.

**Comment by Bisaloo** [comment]
Thank you!

**Comment by pipboy96** [comment]
@Bisaloo Can you go through more of my PRs, going from the least recent preferably? I see the project's activity is kind of slowing down. The reason I ask is that I want to discuss plans on working through the existing issue/PR backlog with zoracon.

**Comment by Bisaloo** [comment]
Sorry, I have very limited time and I mainly review PR that seem important to me (high traffic, sensitive content, website location, a combination of various factors...). I have no plans to change this at the moment.

**Comment by pipboy96** [comment]
@Bisaloo I wanted to specifically work on closing/merging old PRs/issues, but I'm not confident I will be given collaborator access, so I want to ask existing reviewers to put some work into closing old PRs/issues.

**Pull request merged**